src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -596,8 +596,9 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox(
         "Hide viewer count and stream length while hovering over split header",
         s.streamerModeHideViewerCountAndDuration);
-    layout.addCheckbox("Hide moderation actions", s.streamerModeHideModActions,
-                       false, "Hide bans & timeouts from appearing in chat.");
+    layout.addCheckbox(
+        "Hide moderation actions", s.streamerModeHideModActions, false,
+        "Hide bans, timeouts and automod messages from appearing in chat.");
```

Comment:
This is not technically a good description of what this PR did, BUT this setting already did hide automod messages so now the tooltip is more correct than it was before, and automod messages I believe is blanket enough to cover what we changed here.

Reply:
```suggestion
        "Hide bans, timeouts, and automod messages from appearing in chat.");
```
actually oxford would be mad at us if we didn't